pay-api/migrations/versions/b05d371ad030_payment_failed_status.py

```diff
@@ -0,0 +1,46 @@
+"""payment_failed_status
+
+Revision ID: b05d371ad030
+Revises: b7443d501d98
+Create Date: 2020-11-23 11:56:48.721202
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy import String
+from sqlalchemy.sql import column, table
+
+# revision identifiers, used by Alembic.
+revision = 'b05d371ad030'
+down_revision = 'b7443d501d98'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('payment', sa.Column('cons_inv_number', sa.String(length=50), nullable=True))
```

Comment:
is this cons_inv_number in the model?somehow i cant see it..

Reply:
That was from my changes for consolidated invoices, shouldn't impact any existing features.